gcp_variant_transforms/options/variant_transform_options.py

```diff
@@ -597,3 +597,22 @@ def _validate_inputs(parsed_args):
       (not parsed_args.input_pattern and not parsed_args.input_file)):
     raise ValueError('Exactly one of input_pattern and input_file has to be '
                      'provided.')
+
+
+class ExperimentalOptions(VariantTransformsOptions):
+  """Options for experimental features.
+
+  All big releases should be protected behind flags. During the release, the
```

Comment:
I don't think we need this commentary :) This is just what experimental flags are always used for.